content/systems/filecoin_markets/retrieval_market/_index.md

```diff
@@ -9,89 +9,75 @@ weight: 2
 
 ## Components
 
-The `retrieval market` refers to the process of negotiating deals for a provider to serve stored data to a client.
+The `retrieval market` refers to the process of negotiating deals for a provider to serve stored data to a client. It should be highlighted that the negotiation process for the retrieval happens primarily off-chain. It is only some parts of it (mostly relating to redeeming vouchers from payment channels) that involve interaction with the blockchain.
 
 The main components are as follows:
 
 - A payment channel actor (see {{<link payment_channel_actor>}} for details)
-- 2 `libp2p` services - 
+- Two `libp2p` services - 
    - a protocol for making queries
-   - a protocol for negotiating and carrying out retrieval deals (will disappear as a seperate libp2p protocol as retrieval markets evolve)
+   - a protocol for negotiating and carrying out retrieval deals. The intention for this protocol is to become a separate libp2p protocol as the retrieval market design evolves in the near future.
```

Comment:
This is not correct -- there is simply just the data transfer protocol. there is no more libp2p retrieval deal protocol. So there is only the retrieval query protocol.